src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java

```diff
@@ -21,16 +26,11 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         PrintWriter out = response.getWriter();
         out.println("<h1>" + message + "</h1>");
         out.println("<ul>");
-        out.println("<li>Bearchitect</li>");
-        out.println("<li>58050237 Chayapol Limanon</li>");
-        out.println("<li>57050294 Pichavarit Panyokaew</li>");
-        out.println("<li>58050276 Thanathat Surakhup</li>");
-        out.println("<li>58050309 Pasthiyakan Kaewpitak</li>");
-        out.println("<li>58050355 Panuwich Laoudom</li>");
-        out.println("<li>57050269 Patipon Taweechat</li>");
-        out.println("<li>57050279 Piyapon Suntikan</li>");
-        out.println("<li>57050356 Apiwoot Narunatwattana</li>");
-        out.println("<li>57050252 Nontakhon kerdmongkol</li>");
+
+        // Change printing algorithm to use NamePrinterLoader instead.
+        for (NamePrinter namePrinter : NamePrinterLoader.getInstance())
```

Comment:
why you use the NamePrinter which use PrintWriter instead of normal String?
why you put the complication to your code?
how can your friend understand and follow up this structure?

Reply:
OK I'll add comment to describe more about the objective to use NamePrinter instead of hard-coded.

Reply:
ถ้ามี Legacy Project ที่นายเข้าไปดู
แล้วมี code กะมี comments นายจะเชื่อ code หรือ comments ???

ให้ code มัน describe ตัวมันนะ ถ้าจำเป็นจริงๆค่อยใช้ comment

Reply:
1. better to put the for loop inside the NamePrinterLoader's method
2. May rename NamePrintWriterLoader to NameLoader
then it will be
`NameLoader
.getInstance()
.printAllName(out)`